app/models/program.rb

```diff
@@ -27,7 +27,7 @@ def confirmed
     end
 
     def scheduled
-      where.not(start_time: nil)
+      where.not(start_time: nil).where.not(room: nil)
```

Comment:
I guess the same way we currently have the tours scheduled with time, but without room. (which is, I think, what made XML export fail)